StreamNode/WebClient/src/routes/dashboard/index.js

```diff
@@ -138,6 +150,15 @@ class Dashboard extends Component {
         this.setState({deviceFilter: device});
     }
 
+    onDroppedEvent(fromIndex, toIndex){
+        if(fromIndex === toIndex) return;
+        const data = [...this.state.volumes];
+        const item = data.splice(fromIndex, 1)[0];
```

Comment:
Take a look how you get item from data array, maybe you only need  data[fromIndex]